### PR TITLE
feat: 🎸 Edit saved query - set visualization in metadata

### PR DESCRIPTION
**Pull request by mczapkowicz**

### CODE REVIEW

File: src/components/ChartWidgetEditor/components/DataViz/Dataviz.tsx [hunk 1]
```diff
@@ -14,6 +10,8 @@ import getChartInput from '../../../../utils/getChartInput';
 import { CONTAINER_ID } from './constants';
 import { DashboardSettings } from '../../../../modules/dashboards';
 
+import { WidgetSettings, Tag } from '@keen.io/widgets';
```
Comment (apryka) [inline src/components/ChartWidgetEditor/components/DataViz/Dataviz.tsx:13]: what's the difference between `WidgetSettings` from `@keen.io/widget-picker` and `@keen.io/widgets`?

can we unify them?

File: src/modules/chartEditor/saga/restoreSavedQuery.test.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import { Query } from '@keen.io/query';
+import sagaHelper from 'redux-saga-testing';
+import { getContext, put, select } from 'redux-saga/effects';
+import { PUBSUB } from '../../../constants';
+import { chartEditorSelectors } from '../selectors';
+import { chartEditorActions } from '../index';
+import {
+  SET_CHART_SETTINGS,
+  SET_QUERY_EVENT,
+} from '@keen.io/query-creator/dist';
```
Comment (apryka) [inline src/modules/chartEditor/saga/restoreSavedQuery.test.ts:10]: is dist required here?

Comment (apryka) [reply src/modules/chartEditor/saga/restoreSavedQuery.test.ts:10]: please organize your imports

for example
- external sources
- @keen.io
- utils
- constants

File: src/modules/chartEditor/saga/restoreSavedQuery.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import { PUBSUB } from '../../../constants';
+import { getContext, put, select } from 'redux-saga/effects';
+import { chartEditorActions, chartEditorSelectors } from '../index';
```
Comment (apryka) [inline src/modules/chartEditor/saga/restoreSavedQuery.ts:3]: import them from folder

File: src/modules/chartEditor/saga/restoreSavedQuery.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import {
+  SET_CHART_SETTINGS,
+  SET_QUERY_EVENT,
+} from '@keen.io/query-creator/dist';
```
Comment (apryka) [inline src/modules/chartEditor/saga/restoreSavedQuery.ts:7]: no dist please

File: src/components/ChartWidgetEditor/components/DataViz/Dataviz.tsx (hunk 1 above)
Comment (maciejrybaniec) [reply src/components/ChartWidgetEditor/components/DataViz/Dataviz.tsx:13]: WidgetSettings from widget-picker represents only subset of settings that could be defined in picker component. WidgetSettings from `@keen.io/widgets` contains all widget settings.

**Review by maciejrybaniec — APPROVED** [review]

**Pull request merged**